src/NBitcoin/Networks.cs

```diff
@@ -315,43 +315,45 @@ private static Network InitReg()
 
         private static Network InitStratisMain()
         {
-            var consensus = new Consensus();
-
-            consensus.NetworkOptions = new NetworkOptions();
+            Network network = new Network();
+            network.Name = "StratisMain";
+            network.RootFolderName = StratisRootFolderName;
+            network.DefaultConfigFilename = StratisDefaultConfigFilename;
 
-            consensus.SubsidyHalvingInterval = 210000;
-            consensus.MajorityEnforceBlockUpgrade = 750;
-            consensus.MajorityRejectBlockOutdated = 950;
-            consensus.MajorityWindow = 1000;
-            consensus.BuriedDeployments[BuriedDeployments.BIP34] = 227931;
-            consensus.BuriedDeployments[BuriedDeployments.BIP65] = 388381;
-            consensus.BuriedDeployments[BuriedDeployments.BIP66] = 363725;
-            consensus.BIP34Hash = new uint256("0x000000000000024b89b42a942fe0d9fea3bb44ab7bd1b19115dd6a759c0808b8");
-            consensus.PowLimit = new Target(new uint256("00000fffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"));
-            consensus.PowTargetTimespan = TimeSpan.FromSeconds(14 * 24 * 60 * 60); // two weeks
-            consensus.PowTargetSpacing = TimeSpan.FromSeconds(10 * 60);
-            consensus.PowAllowMinDifficultyBlocks = false;
-            consensus.PowNoRetargeting = false;
-            consensus.RuleChangeActivationThreshold = 1916; // 95% of 2016
-            consensus.MinerConfirmationWindow = 2016; // nPowTargetTimespan / nPowTargetSpacing
+            network.Consensus.NetworkOptions = new NetworkOptions();
```

Comment:
All "network.Consensus... = ..." could be moved to a "StratisConsensus" class, derived from Consensus.

Reply:
Yep but what I meant is that that line plus all the following ones of the form `network.Consensus... = ...` could be moved to a separate class - e.g. StratisConsensus. Not something we need to do right now though.